README.md

```diff
@@ -540,10 +542,10 @@ TODO: describe me.
 ## Examples
 -----------
 
-  The [examples/connect.lua](../examples/connect.lua) file shows how to connect 
+  The [examples/connect.lua](/examples/connect.lua) file shows how to connect 
```

Comment:
Ouch, missed that `/examples/...` here. Should be `examples/...` — relative URL.